specificatie/hypotheken.yaml

```diff
@@ -144,6 +144,9 @@ components:
           items:
             $ref: "domain.yaml#/components/schemas/PersoonBeperkt"
             description: "Geldverstrekker die een recht van hypotheek vestigt als zekerheid voor de lening."
+        bedragZekerheidsstelling:
+          $ref: "domain.yaml#/components/schemas/Bedrag"
+          description: "Het bedrag waarvoor hypothecaire zekerheid is gegeven. De vermelde hoofdsom is het bedrag waarvoor het recht van Hypotheek is gevestigd"
```

Comment:
het bedrag dat de hypotheekgever (de geldlener) heeft gekregen, en waarvoor de hypothecaire zekerheid is gegeven aan de geldverstrekker. 

@HildeVos, klopt dat?

Reply:
Deze omschrijving klopt volgens mij niet. In het IMKAD staat de volgende omschrijving :  

_Zekerheidsstelling hypotheek is op 1 of meer kadastrale objecten verleend ter grootte van Bedrag.

Het bedrag waarvoor hypothecaire zekerheid is gegeven. De vermelde hoofdsom is het bedrag waarvoor het recht van Hypotheek is gevestigd (dus niet het bedrag van de geldlening)._

Je kan er voor kiezen om een hogere zekerheidsstelling in te schrijven bij het kadaster dan je aan geldlening krijgt. Als je dat doet (in overleg met de bank) dan kan je eventueel later de geldlening verhogen zonder dat je daarvoor naar de notaris hoeft te gaan om de zekerheidsstelling  te verhogen.

Reply:
Aha! Thnx Johan!

Nog een poging: 
"Het bedrag waarvoor hypothecaire zekerheid is gegeven door onderpandgever en waarvoor het recht van Hypotheek is gevestigd".